librz/core/cmd_analysis.c

```diff
@@ -2908,22 +2908,27 @@ RZ_IPI int rz_cmd_analysis_fcn(void *data, const char *input) {
 			case 'a': // "aflsa"
 				core->analysis->fcns->sorted = false;
 				rz_list_sort(core->analysis->fcns, cmpaddr);
+				rz_core_analysis_fcn_list(core, NULL, "o");
```

Comment:
why `"o"`?

Reply:
I think it can jump to the default branch in `rz_core_analysis_fcn_list` as same as the command `afl`.

Reply:
Ok, good enough for now! That `o` does not have any particular meaning, but considering it's already used elsewhere it is fine for now, thanks!